src/main/java/org/apache/sling/feature/cpconverter/cli/ContentPackage2FeatureModelConverterLauncher.java

```diff
@@ -104,6 +104,9 @@
     @Option(names = { "--entry-handler-config" }, description = "Config for entry handlers that support it (classname:<config-string>", required = false)
     private List<String> entryHandlerConfigs = null;
 
+    @Option(names = { "--remove-install-hooks" }, description = "", required = false)
```

Comment:
Should this remove internal hooks as well? That has not happened in the past...

Reply:
We should treat internal and external hooks the same

Reply:
Done in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/62/commits/3a394314ad168698ba58f8cc05a44a4c09b1d9e2